src/util/file_test.go

```diff
@@ -62,32 +50,60 @@ func cleanup(fn string) {
 	os.Remove(fn + ".bak")
 }
 
-func TestInitDataDir(t *testing.T) {
-	defer os.RemoveAll("./.test")
-	d := "./.test/test"
-	assertDirNotExists(t, d)
-	dir := InitDataDir(d)
-	assertDirExists(t, dir)
-	_, err := os.Stat(dir)
-	assert.Nil(t, err)
-	os.RemoveAll(dir)
+func resetDataDir() {
+	DataDir = ""
 }
 
-func TestInitDataDirDefault(t *testing.T) {
-	defaultDataDir := ".skycointestXCAWDAWD232232"
-	home := UserHome()
-	assertDirNotExists(t, filepath.Join(home, defaultDataDir))
-	dir := InitDataDir(defaultDataDir)
-	assert.NotEqual(t, dir, "")
-	assert.True(t, strings.HasSuffix(dir, defaultDataDir))
-	assertDirExists(t, dir)
-	os.RemoveAll(dir)
+func TestBuildDataDir(t *testing.T) {
+	require.Empty(t, DataDir)
+	defer resetDataDir()
+
+	dir := "./.test-skycoin/test"
```

Comment:
These tests do not create a folder anymore, to avoid needing removal.  So any value for `dir` is ok (except the banned values like `./`).